pkg/polaris/ctl_polaris.go

```diff
@@ -120,45 +120,64 @@ func (ctl *CRSpecBuilderFromCobraFlags) SetPredefinedCRSpec(specType string) err
 // The flags map to fields in the CRSpecBuilderFromCobraFlags struct.
 // master - if false, doesn't add flags that all Users shouldn't use
 func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Command, master bool) {
-	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris")
-	cmd.Flags().StringVar(&ctl.EnvironmentDNS, "environment-dns", ctl.EnvironmentDNS, "Environment DNS")
-	cmd.Flags().StringVar(&ctl.ImagePullSecrets, "pull-secret", ctl.ImagePullSecrets, "Pull secret")
-	cmd.Flags().StringVar(&ctl.StorageClass, "storage-class", ctl.StorageClass, "Set the storage class to use for all the PVC")
-	cmd.Flags().BoolVar(&ctl.EnableReporting, "enable-reporting", GetPolarisDefault().EnableReporting, "Send this flag if you wish to enable ReportingPlatform.")
-	cmd.Flags().StringVar(&ctl.GCPServiceAccount, "gcp-service-account-path", ctl.GCPServiceAccount, "Google Cloud Service account")
-	cmd.Flags().StringVar(&ctl.IngressClass, "ingress-class", GetPolarisDefault().IngressClass, "The name of the ingress class to use.")
-	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Docker registry e.g. docker.io/myuser")
 
-	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", ctl.PostgresHost, "Postgres Host")
-	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", ctl.PostgresPort, "PostgresPort")
-	cmd.Flags().StringVar(&ctl.PostgresUsername, "postgres-username", ctl.PostgresUsername, "Postgres username")
-	cmd.Flags().StringVar(&ctl.PostgresPassword, "postgres-password", ctl.PostgresPassword, "Postgres password")
-	cmd.Flags().StringVar(&ctl.PostgresSSLMode, "postgres-ssl-mode", string(GetPolarisDefault().PolarisDBSpec.PostgresDetails.SSLMode), "Postgres ssl mode [disable|require]")
-	cmd.Flags().BoolVar(&ctl.PostgresInternal, "postgres-container", GetPolarisDefault().PolarisDBSpec.PostgresDetails.IsInternal, "Use in-cluster postgres in a container (Not recommended)")
+	// [DEV NOTE:] please organize flags in order of importance
+	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris")
```

Comment:
Are these changes put into the ctl_polairs_test file?

Reply:
I just realized there is no test file. Never mind... for now. (I think we should make one)

Reply:
Hence, my TODO note on unit testing 😄